lr_face/models.py

```diff
@@ -120,6 +147,8 @@ def embed(self,
             # If the embedding has not been cached to disk yet: compute the
             # embedding, cache it afterwards and then return the result.
             embedding = self.model.predict(x)[0]
+            # Normalize embeddings for finetuning.
+            embedding = embedding / np.linalg.norm(embedding)
```

Comment:
Finetuning doesn't use the `embed()` method, so the models themselves should output normalized embeddings. You can't rely on post-processing for that.